src/main/java/com/aws/greengrass/lifecyclemanager/KernelLifecycle.java

```diff
@@ -265,94 +266,55 @@ void initConfigAndTlog(String configFilePath) {
     void initConfigAndTlog() {
         try {
             Path transactionLogPath = nucleusPaths.configPath().resolve(Kernel.DEFAULT_CONFIG_TLOG_FILE);
-            boolean readFromNonTlog = false;
+            boolean readFromTlog = false;
```

Comment:
Should the default be true, now that this boolean is the opposite of `readFromNonTlog` ?

Reply:
will update